### PR TITLE
Feature/config dev webpack

### PR DESCRIPTION
**Pull request by HuyAms**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,17 +4,22 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "webpack --config webpack/webpack.dev.js",
+    "start": "webpack-dev-server --config webpack/webpack.dev.js",
```
Comment (vinhlee95) [inline package.json]: Should we add ` --open` here for open devServer instantly?

File: webpack/webpack.dev.js [hunk 2]
```diff
@@ -12,6 +29,21 @@ module.exports = merge(common, {
 
   module: {
     rules: [
+      {
+        test: /\.(?:sa|sc|c)ss$/,
```
Comment (vinhlee95) [inline webpack/webpack.dev.js]: Is this needed for production as well? If it is then it should be in webpack.common.js

File: webpack/webpack.dev.js [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
+
+  output: {
+    filename: '[name].js',
+    chunkFilename: '[name].chunk.js'
+  },
+
+  devServer: {
+    contentBase: path.resolve(__dirname, '../public/index.html'),
+    port: 3000,
+    open: true,
```
Comment (HuyAms) [inline webpack/webpack.dev.js]: @vinhlee95 we've already set 'open' here, so no need to add it to script

File: webpack/webpack.dev.js (hunk 2 above)
Comment (HuyAms) [reply webpack/webpack.dev.js]: In product, I intend to use mini-extract-css while using style-loader

File: webpack/webpack.dev.js (hunk 3 above)
Comment (vinhlee95) [reply webpack/webpack.dev.js]: Nice

**Pull request merged**